cmake/test/gtest.cmake

```diff
@@ -226,16 +227,19 @@ endfunction()
 # :param[out] gmock_main_libs: GMock's main libraries
 # :param[out] base_dir: The base directory containing Google Test and/or GMock CMakeLists.txt
 #
-function(catkin_find_google_test_source include_path src_path
+function(catkin_find_google_test_source gtest_path googletest_path
   gtest_found gtest_include_dir gtest_lib_dir gtest_libs gtest_main_libs
   gmock_found gmock_include_dir gmock_lib_dir gmock_libs gmock_main_libs
   base_dir
 )
-  #Find GTest
-  set(_gtest_include_paths "${include_path}/gtest")
-  set(_gtest_source_paths "${src_path}/gtest/src")
-  list(APPEND _gtest_include_paths "${include_path}/googletest/googletest/include/gtest")
-  list(APPEND _gtest_source_paths "${src_path}/googletest/googletest/src")
+  # Path to gtest from the libgtest-dev Debian package.
+  set(_gtest_include_paths "${gtest_path}/include/gtest")
+  set(_gtest_source_paths "${gtest_path}/src/gtest/src")
+
+  # Path to gtest from the googletest Debian package.
+  list(APPEND _gtest_include_paths "${googletest_path}/googletest/include/gtest")
+  list(APPEND _gtest_source_paths "${googletest_path}/googletest/googletest/src")
```

Comment:
The double "googletest" seems to be a mistake. The `include` and `src` directories should be siblings. See #1088.